kind.yaml

```diff
@@ -12,23 +12,24 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - go
+      - go~1.22
```

Comment:
why do we need to pin to specific version of go 
```suggestion
      - go
```

Reply:
Removing specific version, cleaning this up